docs/src/reference/experimental/loop-contracts.md

```diff
@@ -29,6 +29,9 @@ Kani needs to unwind the loop for `u64::MAX` iterations, which is computationall
 With loop contracts, we can specify the loop’s behavior using invariants. For example:
 
 ``` Rust
+#![feature(stmt_expr_attributes)]
```

Comment:
Are these necessary to include here?

Reply:
It is necessary for use attribute on `stmt_expr` like loops.